guides/BACKEND.md

```diff
@@ -16,6 +16,9 @@ Before you can start the Docker containers, you must run the following.
 # Copy the example environment
 cp .env.sample .env
 
+# Add a github token from https://github.com/settings/tokens to the .env
+echo "GITHUB_API_TOKEN=<github-token>" >> .env
```

Comment:
This code snippet isn't terribly useful as we already have `GITHUB_API_TOKEN=` prefilled

Reply:
Cool. I'm commenting out the line from .env.sample
So that a new builder can learn from either place and add the token.